compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/coneDiagnosticToFirDiagnostic.kt

```diff
@@ -89,24 +89,36 @@ private fun mapUnsafeCallError(
 
     val candidateFunction = candidate.symbol.fir as? FirSimpleFunction
     val candidateFunctionName = candidateFunction?.name
-    val left = candidate.callInfo.explicitReceiver
-    val right = candidate.callInfo.argumentList.arguments.singleOrNull()
-    if (left != null && right != null &&
+    val receiverExpression = candidate.callInfo.explicitReceiver
+    val singleArgument = candidate.callInfo.argumentList.arguments.singleOrNull()
+    if (receiverExpression != null && singleArgument != null &&
         source.elementType == KtNodeTypes.OPERATION_REFERENCE &&
         (candidateFunction?.isOperator == true || candidateFunction?.isInfix == true)
     ) {
         val operationToken = source.getChild(KtTokens.IDENTIFIER)
         if (candidateFunction.isInfix && operationToken?.elementType == KtTokens.IDENTIFIER) {
-            return FirErrors.UNSAFE_INFIX_CALL.on(source, left, candidateFunctionName!!.asString(), right)
+            return FirErrors.UNSAFE_INFIX_CALL.on(
+                source,
+                receiverExpression,
+                candidateFunctionName!!.asString(),
+                singleArgument,
+                receiverExpression
```

Comment:
Don't understand why it's necessary to pass receiver expression twice here

Reply:
Thanks! Let me remove `lhs` and `rhs` and rename them to `receiverExpression` and `argumentExpression` since `lhs` and `rhs` can be wrong for `in` operator.

Reply:
Ok